packages/components/src/animations/pop-in.ts

```diff
@@ -0,0 +1,19 @@
+const DEFAULT_DURATION = 250;
+const DEFAULT_EASING = 'cubic-bezier(0.34, 1.56, 0.64, 1)';
```

Comment:
Can you use the same timing function as previously declared in the `pop-in-animation`? 
Here you are using a `cubic-bezier` intead of a `linear` easing.

To know more about timing functions (also called easing), here is the documentation: https://developer.mozilla.org/en-US/docs/Web/CSS/transition-timing-function